modules/mongodb/build.gradle

```diff
@@ -0,0 +1,41 @@
+description = "Testcontainers :: MongoDB"
+
+dependencies {
+    compile project(':testcontainers')
+
+    testCompile("org.junit.jupiter:junit-jupiter-api:5.5.1")
+    testCompile('org.mockito:mockito-core:3.0.0')
+    testCompile('ch.qos.logback:logback-classic:1.2.3')
+    testRuntime("org.junit.jupiter:junit-jupiter-engine:5.5.1")
+    testCompile("org.mongodb:mongodb-driver-reactivestreams:1.12.0")
+    testCompile("org.mongodb:mongodb-driver-sync:3.11.0")
+    testCompile("org.mockito:mockito-junit-jupiter:3.0.0")
+}
+
+tasks.withType(Test) { Test task ->
```

Comment:
please follow other modules' test configuration for consistency

Reply:
done